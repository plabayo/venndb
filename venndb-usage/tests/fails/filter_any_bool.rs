use venndb::VennDB;

#[derive(Debug, VennDB)]
struct Employee {
    #[venndb(filter, any)]
    is_alive: bool,
}

fn main() {}
