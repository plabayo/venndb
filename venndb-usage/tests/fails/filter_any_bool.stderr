error: Invalid field-level `venndb` attribute
       `any` cannot be used with `bool`
 --> tests/fails/filter_any_bool.rs:6:15
  |
6 |     is_alive: bool,
  |               ^^^^
